pages/api/content/index.js

```diff
@@ -1,5 +1,32 @@
 import sql from "~/lib/postgres";
 
-const handler = async (req, res) => res.status(200).json({ data: await sql`SELECT * FROM content;` });
+const handler = async (req, res) => {
+	if (req.method === "GET") {
+		return res.status(200).json({ data: await sql`SELECT * FROM content;` });
+	}
+	if (req.method === "POST") {
+		const { title, content_type, description, issueID, img_url, video_url } = req.body;
+
+		const checkContentTitle = await sql`SELECT * FROM content WHERE title=${title};`;
+
+		const checkIssueID = await sql`SELECT * FROM issues WHERE id=${issueID};`;
```

Comment:
is issueID required? Can we create content without an issue id? Maybe i'm not sure what issue it belongs to yet, or it belongs to many.

Reply:
removed it